.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
I may have encountered this issue before, so here's a guess.
If the purpose of this underscore is to please editorconfig, then please convince editorconfig otherwise, instead, by editing `.editorconfig`, adding a section for this file specifically, setting it to not have a closing newline.

Reply:
Husky `v5` add a `_` folder with a `sh` script, and `husky` ignore it.
I didn't create this `.gitignore`, it is automatically managed/generated by husky since `v5`.

Reply:
Oh, I see.